test/testHelpers.ts

```diff
@@ -36,20 +28,19 @@ export function readMockUFCResponse(filename: string): {
 }
 
 export function readAssignmentTestData(): IAssignmentTestCase[] {
-  const testCaseData: IAssignmentTestCase[] = [];
-  const testCaseFiles = fs.readdirSync(ASSIGNMENT_TEST_DATA_DIR);
-  testCaseFiles.forEach((file) => {
-    const testCase = JSON.parse(fs.readFileSync(ASSIGNMENT_TEST_DATA_DIR + file, 'utf8'));
-    testCaseData.push(testCase);
-  });
-  return testCaseData;
+  return fs
+    .readdirSync(ASSIGNMENT_TEST_DATA_DIR)
+    .map((file) => JSON.parse(fs.readFileSync(ASSIGNMENT_TEST_DATA_DIR + file, 'utf8')));
 }
 
 export function getTestAssignments(
   testCase: IAssignmentTestCase,
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  assignmentFn: any,
-  obfuscated = false,
```

Comment:
we do need this flag to make sure we run both obfuscated and non-obfuscated tests, right?

Reply:
it looks like the obfuscated argument is just getting dropped since the assignment functions only take 4 params.

<img width="1212" alt="Screenshot 2024-06-06 at 1 15 04 PM" src="https://github.com/Eppo-exp/js-client-sdk-common/assets/1248641/38f80559-54d0-4f4d-8a2f-4711e6914862">

Reply:
Ok yes good catch -- a recent change moved the obfuscation parameter to the client initialization and indeed this can be removed. Good catch!

Reply:
greg is right, thanks for clarifying